EZSwiftExtensionsTests/DateTests.swift

```diff
@@ -73,7 +73,7 @@ class DateTests: XCTestCase {
 
     func testDateToString() {
         let date = Date(timeIntervalSince1970: 0)
-
+        let format = "yyyy-MM-dd"
```

Comment:
Could you add tests here to verify the size of the dictionary increasing and decreasing as needed. We want to validate at unit level that the right objects were indeed cached.

Reply:
I have updated the pull request kindly review @Khalian

Reply:
Unit test added to test the `dict` functionality.